js/security.js

```diff
@@ -45,13 +45,18 @@ exports = module.exports = function (config, sri4nodeUtils) {
           if (op.retry(err)) {
             return;
           }
-
+          console.warn('SECURITY REQUEST FAILED: ' + err);
           promise.reject(err);
         }
 
         if (response.statusCode === 200) {
           promise.resolve(response.body);
         } else {
+          if(response.body){
+            console.warn('SECURITY REQUEST FAILED: ' + response.statusCode + ' - ' + JSON.stringify(response.body));
```

Comment:
Is this still necessary or was it just for debugging?
Should this file be changed at all?